docs-ui/components/button.stories.js

```diff
@@ -33,7 +33,19 @@ storiesOf('Buttons', module)
         </Item>
 
         <Item>
-          <Button to={''} disabled onClick={action('click disabled')}>
+          <Button to={'#href'} disabled onClick={action('click disabled')}>
+            Disabled Button
+          </Button>
+        </Item>
+
+        <Item>
+          <Button href="#href" disabled onClick={action('click disabled')}>
```

Comment:
Added some different disabled Button types to storybook to check that nothing happens when you click a disabled Button